File: fizzbuzz/fizzbuzz.cs
using System;
					
public class Program
{
	public static void Main()
	{ 

    for(int loop = 1; loop <= 100; loop++){

			if (loop%3==0 && loop%5==0){
				Console.WriteLine("FizzBuzz");
			}
			else if(loop%3==0){
				Console.WriteLine("Fizz");
			}
			else if(loop%5==0){
				Console.WriteLine("Buzz");
			}
		else{
    		Console.WriteLine(loop);
		    }
		}
    }
}

File: fizzbuzz/fizzbuzz.r
for(val in 1:100)
{
    if((val%%3==0) || (val%%5==0)){
        if(val%%3==0 && val%%5==0){
            print("FizzBuzz")
        }
        else if(val%%3==0 && !val%%5==0){
            print("Fizz")
        }
        else if(val%%5==0 &&  !val%%3==0){
            print("Buzz")
        }
    }
    else{
        print(val)
    }
    
}